Doc/library/datetime.rst

```diff
@@ -317,6 +317,10 @@ Notes:
   >>> print(_)
   -1 day, 19:00:00
 
+(6)
+   This isn't quite equivalent to timedelta + (-timedelta), because -timedelta in
+   isolation can overflow in cases where timedelta - timedelta does not.
```

Comment:
Which _timedelta_ is which. Wouldn’t it make more sense to keep using _t2_ and _t3_?

Reply:
I'm not sure it makes much difference which one is which:

The issue is that `t1 - t2 == t1 + (-t2)` for all cases *except* when `t2` is `timedelta.max`, in which case the right side of the equation overflows, because `(-timedelta)` is only valid on the interval `(-timedelta.max, timedelta.max]`. This is why `timedelta + (-timedelta)` and `timedelta - timedelta` are not equivalent.

On the other hand, if you interpret it as Python code `timedelta + (-timedelta)` would suggest the addition of the `type` objects themselves, and I don't think it would hurt to switch over to `t1`, `t2` and `t3` to make it clear that these are specific instances.

@abalkin Thoughts?

Reply:
I agree that it best to follow the notation used in the body of the table in footnotes.  How is this wording?

> The expression ` t1 - t2` is not always the same as `t1 + (-t2)`.  In the case when `t2` is `timedelta.max`, `(-t2)` in the second expression will overflow, while the first expression will produce an answer.

Reply:
This actually makes things clearer, let me make this change and see how it looks.